docs/tutorials/cloudflare.md

```diff
@@ -105,7 +105,7 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: external-dns
-  namespace: default
+  namespace: default # NOTE: Change this if you're not in the default namespace.
```

Comment:
Should this line just be removed instead, so that it points to whatever namespace kubectl is targeting (like the rest of the resources in this yaml)?